src/mame/ensoniq/esqpanel.cpp

```diff
@@ -679,28 +713,154 @@ esqpanel2x40_device::esqpanel2x40_device(const machine_config &mconfig, const ch
 
 void esqpanel2x40_vfx_device::device_add_mconfig(machine_config &config)
 {
-	ESQ2X40(config, m_vfd, 60);
+	ESQ2X40_VFX(config, m_vfd, 60);
+
+	if (m_panel_type == VFX)
+		config.set_default_layout(layout_vfx);
+	else if (m_panel_type == VFX_SD)
+		config.set_default_layout(layout_vfxsd);
+	else if (m_panel_type == SD_1 || m_panel_type == SD_1_32)
+	  config.set_default_layout(layout_sd1);
```

Comment:
something went wrong with the indentation just there

Reply:
I shall do that!